manifests/params.pp

```diff
@@ -14,7 +14,8 @@
 
   $context = 'i2b2',
   $local_base_url = 'http://localhost:8080',
-  $external_base_url = "http://${::fqdn}:8080",
+  $external_hostname = $::fqdn,
+  $external_base_url_tomcat = "http://${::fqdn}:8080",
```

Comment:
AFAIK, this is used to build the URLs that the PM advertises. I don't see anything changing such a default so that TLS is used.